### ISSUE TITLE
Screen is broken when installed in or moved to a subfolder of the programs folder

### ISSUE THREAD

**Issue by RubbrDug** [mixxxdj/mixxx#13951]
### Bug Description

As in the title. 

### Version

2.4.2

### OS

macOS 15.1.1 ARM

**Comment by Swiftb0y** [comment]
Please describe what you did in more detail? Which "Screen"? Are you referring to the main window? What exactly did you move in what exact folder?

**Comment by ronso0** [comment]
There are no images being transmitted if you reply via email.
Please edit your post and insert the images. Thak you!

**Comment by RubbrDug** [comment]
Thank you for being so patient with me. I added the screenshots.

**Comment by ronso0** [comment]
I'm not quite sure what exactly you did, but it looks like icon paths (set in skins/LateNight/style_palemoon.qss) can't be resolved, while the fader graphics for example (set in xml templates) are found.

> This is mixxx after moving it into program files/audio (where I keep my audio related software):

From the window decorations I assume you're on macOS.
Usually/often relevant app paths are hardcoded (resources like styles, translations etc.), so manually moving app components around on your disk and expect an app to tolerate that is rather ... optimistic.

**Comment by RubbrDug** [comment]
Yes, I am on MacOS (I gave information about it in the initial post). Moving apps around in the program folder and his subdirectories is usually no problem and encouraged to organize your apps. It also happens when you directly install to a subfolder. This is what happened to me. I can live with this, I just thought someone might be interested in optimizing the app.

 

Von: ronso0 ***@***.***>
Antworten an: mixxxdj/mixxx ***@***.***>
Datum: Dienstag, 3. Dezember 2024 um 11:12
An: mixxxdj/mixxx ***@***.***>
Cc: RubbrDug ***@***.***>, Author ***@***.***>
Betreff: Re: [mixxxdj/mixxx] Screen is broken when installed in or moved to a subfolder of the programs folder (Issue #13951)

 

I'm not quite sure what exactly you did, but it looks like icon paths (set in skins/LateNight/style_palemoon.qss) can't be resolved, while the fader graphics for example (set in xml templates) are found.

This is mixxx after moving it into program files/audio (where I keep my audio related software):

From the window decorations I assume you're on macOS.
Usually/often relevant app paths are hardcoded (resources like styles, translations etc.), so manually moving app components around on your disk and expect an app to tolerate that is rather ... optimistic.

**Comment by ronso0** [comment]
Hmkay, would you mind sharing the (official) resources that encourages to move apps?

Btw when replying via email please avoid quoting the message you respond to, otherwise conversations here get rather cluttered. Thank you.
